EAD-Session4/src/com/metacube/Parking/controller/AddPlan.java

```diff
@@ -0,0 +1,33 @@
+/**
+* This Servlet is a controller to add plans
+* @author Abhishek Bishnoi
+* @since Aug 29,2019
+*/
+package com.metacube.Parking.controller;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.metacube.Parking.dao.PlanDao;
+import com.metacube.Parking.model.Plan;
+import com.metacube.Parking.utility.SelectPlanHtml;
+/**
+*	Servlet to Add plan to db
+*/
+public class AddPlan extends HttpServlet{
+	public AddPlan(){
+		super();
+	}
+	protected void service(HttpServletRequest request, HttpServletResponse response) throws IOException{
```

Comment:
We should name the method, 'service' is not  enough. If there are multiple methods in a single controller, how we will identify, the functionality of each 'service', if we do not have a proper name ?